src/api/api.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+import axios from 'axios';
```

Comment:
- [ ] Kindly enable the linter in this file. Linter enforces quality code standards and therefore should not be dissabled.